pkg/controller/certificates/certificate_controller_utils.go

```diff
@@ -25,6 +25,8 @@ func IsCertificateRequestApproved(csr *certificates.CertificateSigningRequest) b
 	return approved && !denied
 }
 
+// GetCertApprovalCondition returns two booleans, the first is true if the
```

Comment:
"GetCertApprovalCondition checks whether CSR was approved or denied. A return of (false, false) is possible if CSR is still pending."